recipes/clang-bindings-python/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "5.0.0" %}
+
+package:
+  name: clang-bindings-python
+  version: {{ version }}
+
+source:
+  url: http://llvm.org/releases/{{ version }}/cfe-{{ version }}.src.tar.xz
+  sha256: 019f23c2192df793ac746595e94a403908749f8e0c484b403476d2611dd20970
+
+build:
+  number: 0
+  noarch: python
+  script:
+    - mkdir -p "${SP_DIR}"
+    - cp -r bindings/python/clang "${SP_DIR}"
```

Comment:
This is a re-package, right? Is it safe to make it noarch?
How hard would it be to build it from source?
(Sorry if the answers are obvious but I have no idea what packaging `clang-bindings-python` entails.)

Reply:
No it's [effectively a single file python module](https://github.com/llvm-mirror/clang/tree/master/bindings/python/clang).

It's included in the clang source code and the [README states](https://github.com/llvm-mirror/clang/tree/master/bindings/python) to use it by setting `PYTHONPATH` but I think copying it to the site packages directory is cleaner.

Reply:
The complicated bit is building `libclang` itself but that is handled by the `clangdev` feedstock.

Reply:
> No it's effectively a single file python module .

Got it. Thanks!